### PR TITLE
[DO-1310] source file fix

### PR DESCRIPTION
**Pull request by Iva-khar**
after 12.30.2020 the full UO source file from data.gov.ua needs to be decoded and replaced some symbols by this method

### CODE REVIEW

File: data_ocean/downloader.py [hunk 1]
```diff
@@ -188,3 +189,13 @@ def update_field(self, dataset_name, field_name, new_field_value):
         setattr(dataset, field_name, new_field_value)
         dataset.save(update_fields=[field_name, 'updated_at'])
 
+    # after 12.30.2020 the full UO source file from data.gov.ua needs to be decoded and replaced some symbols by this
+    # method
+    def replace(self):
```
Comment (roman-tiukh) [inline data_ocean/downloader.py:194]: rename method
and move to UkrDownloader

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Comment by criticus** [comment]
@Iva-khar what's status of this PR?

**Comment by Iva-khar** [comment]
Waiting for the upgrade

File: data_ocean/downloader.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/downloader.py]: Done and added 2 new replaces

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**